kong/runloop/plugin_servers/mp_rpc.lua

```diff
@@ -325,10 +325,17 @@ end
 
 
 function Rpc:handle_event(plugin_name, conf, phase)
-  local instance_id = self.get_instance_id(plugin_name, conf)
-  local _, err = bridge_loop(self, instance_id, phase)
+  local instance_id, err = self.get_instance_id(plugin_name, conf)
+  local _
+  if not err then
+    _, err = bridge_loop(self, instance_id, phase)
+  end
 
   if err then
+    if err == "not ready" then
+      ngx.sleep(0.1)
```

Comment:
`0.1` is not a reasonable value, do we have better choice?

Reply:
if err is `not ready` then just return early and don't handle the event. but make sure there's log, we can emit log in every 5 seconds for example, preferabaly in the `connection_check_timer` loop.

Reply:
I don't want to introduce a too large delay. 0.1 is just a step, not a timeout.